src/SonarScanner.MSBuild.PreProcessor/ProcessedArgs.cs

```diff
@@ -199,6 +199,12 @@ public ProcessedArgs(
         {
             ScanAllAnalysis = true;
         }
+
+        if (AggregateProperties.TryGetProperty(SonarProperties.Sources, out _) || AggregateProperties.TryGetProperty(SonarProperties.Tests, out _))
+        {
+            logger.LogWarning(Resources.WARN_SourcesAndTestsDeprecated);
```

Comment:
I think it should be a UI warning, wdyt?

Reply:
Yes indeed. I missed that. Thanks!